src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -414,27 +403,23 @@ private NuGetAuditMode ParseAuditMode()
         }
 
         // Enum parsing and ToString are a magnitude of times slower than a naive implementation.
-        public static bool ParseEnableValue(string? value, string projectFullPath, ILogger logger)
+        public static bool ParseEnableValue(RestoreAuditProperties? value, string projectFullPath, ILogger logger)
         {
-            // Earlier versions allowed "enable" and "default" to opt-in
-            if (string.IsNullOrEmpty(value)
-                || string.Equals(value, bool.TrueString, StringComparison.OrdinalIgnoreCase)
-                || string.Equals(value, "enable", StringComparison.OrdinalIgnoreCase)
-                || string.Equals(value, "default", StringComparison.OrdinalIgnoreCase))
+            if (value == null)
             {
                 return true;
+
             }
-            if (string.Equals(value, bool.FalseString, StringComparison.OrdinalIgnoreCase)
-                || string.Equals(value, "disable", StringComparison.OrdinalIgnoreCase))
+
+            if (value!.TryParseEnableAudit(out bool result))
```

Comment:
I still don't know why this `!` is needed.  the `if (value == null)` should make it not required?
```suggestion
            if (value.TryParseEnableAudit(out bool result))
```

Reply:
I some version of the changes it was probably necessary. Not anymore though.
Nice catch.